spec/helpers/middleman/menu_spec.rb

```diff
@@ -71,7 +71,7 @@
 
         pricing = menus[2]
         expect(pricing[:label]).to eq('Pricing')
-        expect(pricing[:title]).to eq("Pricing | Sharesight #{locale_obj[:country]}".strip)
+        expect(pricing[:title]).to match(/Pricing | Sharesight(\s\w+)?/)
```

Comment:
Does this not work…?  I thought we had a `locale_obj[:something]` that would fit here..

🤷  fine by me.

Reply:
No. Problem is our titles are 'manually crafted', but this test assumes it's A+B, which it issn't.
Eg.
- Test Assumes its "... Sharesight United States"
- But it actually is "... Sharesight US"